packages/agoric-cli/src/main.js

```diff
@@ -160,39 +160,63 @@ const main = async (progname, rawArgs, powers) => {
       return subMain(installMain, ['install', forceSdkVersion], opts);
     });
 
-  program
-    .command('deploy [script...]')
-    .description(
-      'run a deployment script with all your user privileges against the local Agoric VM',
-    )
-    .option(
-      '--allow-unsafe-plugins',
-      `CAREFUL: installed Agoric VM plugins will also have all your user's privileges`,
-      false,
-    )
-    .option(
-      '--hostport <host:port>',
-      'host and port to connect to VM',
-      '127.0.0.1:8000',
-    )
-    .option(
-      '--need <subsystems>',
-      'comma-separated names of subsystems to wait for',
-      'local,agoric,wallet',
-    )
-    .option(
-      '--provide <subsystems>',
-      'comma-separated names of subsystems this script initializes',
-      '',
-    )
-    .action(async (scripts, cmd) => {
-      const opts = { ...program.opts(), ...cmd.opts() };
-      return subMain(deployMain, ['deploy', ...scripts], opts);
-    });
+  const addRunOptions = cmd =>
+    cmd
+      .option(
+        '--allow-unsafe-plugins',
+        `CAREFUL: installed Agoric VM plugins will also have all your user's privileges`,
+        false,
+      )
+      .option(
+        '--hostport <host:port>',
+        'host and port to connect to VM',
+        '127.0.0.1:8000',
+      )
+      .option(
+        '--need <subsystems>',
+        'comma-separated names of subsystems to wait for',
+        'local,agoric,wallet',
+      )
+      .option(
+        '--provide <subsystems>',
+        'comma-separated names of subsystems this script initializes',
+        '',
+      );
+
+  addRunOptions(
+    program
+      .command('run <script> [script-args...]')
+      .description(
+        'run a script with all your user privileges against the local Agoric VM',
+      ),
+  ).action(async (script, scriptArgs, cmd) => {
+    const opts = { ...program.opts(), ...cmd.opts(), scriptArgs };
+    return subMain(deployMain, ['run', script], opts);
+  });
+
+  addRunOptions(
+    program
+      .command('deploy [script...]')
+      .description(
+        'run multiple scripts with all your user privileges against the local Agoric VM',
+      ),
+  ).action(async (scripts, cmd) => {
+    const opts = { ...program.opts(), ...cmd.opts() };
+    return subMain(deployMain, ['deploy', ...scripts], opts);
+  });
 
   program
     .command('start [profile] [args...]')
-    .description('run an Agoric VM')
+    .description(
+      `\
+start an Agoric VM
+
+agoric start - runs the default profile (dev)
+agoric start dev -- [initArgs] - simulated chain and solo VM
+agoric start local-chain [portNum] -- [initArgs] - local chain
+agoric start local-solo [portNum] [provisionPowers] - local solo VM
```

Comment:
documenting the start profiles is a bit of a drive-by, but a welcome one...

Reply:
have you looked at docopt, btw?

Reply:
> have you looked at docopt, btw?

Took a look now, but it'll take work to integrate.  Maybe later.